content/posts/newsletter-007/index.md

```diff
@@ -409,6 +409,19 @@ February's full weekly devlogs: "This Week In Veloren...":
 
 ## Library & Tooling Updates
 
+### savefile 0.5.0
+[savefile] is a serialization crate used to effortlessly serialize rust crates 
+and enums into a binary format. Anything implementing the ```Write``` trait can
+serialize and deserialize. Savefile also supports first-class versioning 
+support. This version (0.5.0) now provides an [introspection] feature.
+
+For details, see the [github page] or the [savefile documentation]
+
+[savefile]: https://crates.io/crates/savefile
+[savefile documentation]: https://docs.rs/savefile/0.6.1/savefile
+[introspection]: https://docs.rs/savefile/0.6.1/savefile/#introspection
+[github page]: https://github.com/avl/savefile
```

Comment:
Could you change this ID to something a bit more specific (e.g. `savefile-github`)? Markdown link IDs are scoped to the entire document, rather than the section, so I'm wary of them overlapping.

Note that you can do `[Link Text!][the-actual-link-id]` if you want to decouple the phrasing of the text from the ID :)

Reply:
Didn't realize that MD link ids had global scope. Thanks for letting me know!